src/index.ts

```diff
@@ -507,10 +636,65 @@ export function getOptions(options: RokuDeployOptions = {}) {
     return finalOptions;
 }
 
+export function getStagingFolderPath(options?: RokuDeployOptions): string {
+    options = getOptions(options);
+
+    let stagingFolderPath = path.join(options.outDir, '.roku-deploy-staging');
+    stagingFolderPath = path.resolve(stagingFolderPath);
+    return stagingFolderPath;
+}
+
+
+/**
+ * Centralizes getting output zip file path based on passed in options
+ * @param options
+ */
+export function getOutputZipFilePath(options: RokuDeployOptions): string {
```

Comment:
Remove the `: string` type here; typescript can infer this from the return variable.